_maps/shuttles/emergency_luxury.dmm

```diff
@@ -1,1326 +1,1830 @@
-/obj/item/storage/fancy/cigarettes/cigars/havana{
-	pixel_y = 7
-	},
-/turf/open/floor/mineral/titanium/blue,
-/area/shuttle/escape/luxury)
-"bn" = (
-/obj/effect/turf_decal/delivery,
-/obj/structure/closet/crate/engineering/electrical,
-/obj/effect/decal/cleanable/robot_debris,
-/obj/effect/spawner/random/maintenance/eight,
-/turf/open/floor/iron,
-/area/shuttle/escape/luxury)
-"bo" = (
-/obj/effect/turf_decal/delivery,
-/obj/structure/closet/crate/engineering,
+"Av" = (
 /obj/effect/spawner/random/maintenance/eight,
-/turf/open/floor/iron,
-/area/shuttle/escape/luxury)
-"bp" = (
-/obj/effect/turf_decal/delivery,
-/obj/structure/window{
-	dir = 4
-	},
 /obj/structure/closet/crate{
 	icon_state = "crateopen"
 	},
+/obj/effect/turf_decal/delivery,
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/effect/decal/cleanable/vomit/old,
 /turf/open/floor/iron,
-/area/shuttle/escape/luxury)
-"bq" = (
-/obj/item/stack/rods/ten,
-/turf/open/floor/plating,
-/area/shuttle/escape/luxury)
-"br" = (
-/obj/effect/decal/cleanable/robot_debris,
+/area/shuttle/escape)
+"AB" = (
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/effect/decal/cleanable/glass,
 /turf/open/floor/plating,
-/area/shuttle/escape/luxury)
-"bs" = (
+/area/shuttle/escape)
+"AN" = (
+/obj/structure/girder,
+/obj/effect/turf_decal/delivery,
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/structure/grille/broken,
 /turf/open/floor/plating,
+/area/shuttle/escape)
+"AT" = (
+/obj/machinery/shower/directional/south,
+/turf/open/floor/mineral/titanium/white,
 /area/shuttle/escape/luxury)
-"bt" = (
+"Bj" = (
 /obj/structure/window/reinforced{
-	dir = 1;
-	pixel_y = 2
+	dir = 4;
+	layer = 2.9
 	},
-/obj/item/clothing/head/collectable/paper,
-/turf/open/floor/holofloor/beach/water,
+/obj/effect/spawner/random/structure/closet_private,
+/turf/open/floor/iron/white/herringbone,
 /area/shuttle/escape/luxury)
-"bu" = (
-/obj/structure/window/reinforced{
-	dir = 1;
-	pixel_y = 2
+"Bo" = (
+/obj/item/shard,
+/turf/open/floor/iron,
+/area/shuttle/escape)
+"Br" = (
+/turf/open/floor/iron,
+/area/shuttle/escape)
+"Ck" = (
+/obj/machinery/scanner_gate/luxury_shuttle{
+	layer = 2.6
 	},
-/turf/open/floor/holofloor/beach/water,
+/obj/machinery/door/airlock/silver{
+	name = "First Class"
+	},
+/turf/open/floor/carpet/blue,
 /area/shuttle/escape/luxury)
-"bv" = (
-/obj/machinery/computer/crew{
-	dir = 8
+"CQ" = (
+/turf/closed/wall/mineral/titanium,
+/area/shuttle/escape)
+"CZ" = (
+/obj/machinery/light/small/directional/east,
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/effect/decal/cleanable/glass,
+/turf/open/floor/plating,
+/area/shuttle/escape)
+"Dc" = (
+/obj/item/food/spaghetti/boiledspaghetti{
+	pixel_y = 5
 	},
-/turf/open/floor/mineral/diamond,
+/obj/structure/table/wood/fancy/black,
+/turf/open/floor/carpet/red,
 /area/shuttle/escape/luxury)
-"bw" = (
-/obj/structure/window/reinforced{
-	dir = 1;
-	pixel_y = 2
+"Di" = (
+/obj/docking_port/mobile/emergency{
+	dir = 2;
+	name = "Luxurious Emergency Shuttle"
+	},
```

Comment:
Also, one shuttlecode sufferer to another - you'll need to update `width` and `height` here seeing as the shuttle's larger than it was before. `dwidth` may need updating too. [Here](https://tgstation13.org/wiki/Guide_to_mapping#Shuttles)'s the best guide I can link to, but usually I just get strained and suffer with trial and error.

Reply:
> Also, one shuttlecode sufferer to another - you'll need to update `width` and `height` here seeing as the shuttle's larger than it was before. `dwidth` may need updating too. [Here](https://tgstation13.org/wiki/Guide_to_mapping#Shuttles)'s the best guide I can link to, but usually I just get strained and suffer with trial and error.

No, do not do this. This is done automatically and should never have been done circa 2016

Reply:
lmao ratio

Reply:
> No, do not do this. This is done automatically and should never have been done circa 2016

Really? Damn. I'm guessing that's just for emergency shuttles only then considering I've historically had to fuck with the vars on other shuttle types to get them to load.

Reply:
> > No, do not do this. This is done automatically and should never have been done circa 2016
> 
> Really? Damn. I'm guessing that's just for emergency shuttles only then considering I've historically had to fuck with the vars on other shuttle types to get them to load.

No, it's for all shuttles.

Reply:
MAN REALLY?
Since when, I *need* to backport to some other repos that so nobody else suffers, hell yeah.

Reply:
#71220
#71687